docs-kits/kits/Digital Twin Kit/Software Development View/page_interaction-patterns.md

```diff
@@ -109,8 +110,10 @@ autonumber
         CEDC-->>Con: Dataset for submodel(-bundle)
         Con->>PEDC: negotiate for Dataset and retrieve token
         PEDC-->>Con: access token
-        Con->>SM: GET {{submodel-descriptor/href}}/$value
-        SM-->>Con: data
+        Con->>PEDCDP: GET {{submodel-descriptor/href}}/$value
```

Comment:
href is the dataplane url -> maybe something like: 
{{provider-data.plane}}/submodelpath/$value